egs/iwslt22_ta/ST/pruned_transducer_stateless5/train.py

```
@@ -0,0 +1,1290 @@
+            scaler=scaler,
+            rank=rank,
+        )
+
+    logging.info("Done!")
+
+    if world_size > 1:
+        torch.distributed.barrier()
+        cleanup_dist()
+
+
+def display_and_save_batch(
```

Ruff (0.12.2)

Redefinition of unused `display_and_save_batch` from line 75

(F811)